src/c++/perf_analyzer/client_backend/mock_client_backend.h

```diff
@@ -52,7 +52,31 @@ class MockClientStats {
     enum class ReqType {SYNC, ASYNC, ASYNC_STREAM};
 
     struct SeqStatus {
-      bool live = false;
+      std::set<uint64_t> used_seq_ids;
```

Comment:
It seems like `clang-format-6.0` with [these settings](https://github.com/triton-inference-server/client/blob/main/.clang-format) is not being used to format the files from this PR--can that be done?

Reply:
fixed